contrib/dockerize-image

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+
```

Comment:
Can we add `set -e` here to make sure that if something fails, the script doesn't continue barreling forward? :smile: